include/Primitives/RectangularCuboid.hpp

```diff
@@ -0,0 +1,183 @@
+/*
+** EPITECH PROJECT, 2024
+** Raytracer
+** File description:
+** RectangularCuboid
+*/
+
+#include "IPrimitive.hpp"
+
+namespace Primitive {
+
+    class RectangularCuboid : public IPrimitive {
+
+        public:
+
+            /**
+             * @brief Construct a new RectangularCuboid object.
+             *
+             */
+            RectangularCuboid();
+
+            /**
+             * @brief Construct a new RectangularCuboid object.
+             *
+             * @param maxX double max x of the rectangular cuboid
+             * @param maxY double max y of the rectangular cuboid
+             * @param maxZ double max z of the rectangular cuboid
+             * @param minX double min x of the rectangular cuboid
+             * @param minY double min y of the rectangular cuboid
+             * @param minZ double min z of the rectangular cuboid
+             */
```

Comment:
Did you forget to explain this param: std::shared_ptr<Material::IMaterial> material ?